apps/dashr-yield-curve/app.R

```diff
@@ -0,0 +1,428 @@
+library(plotly)
+library(dplyr)
+library(tidyr)
+library(data.table)
+library(purrr)
+library(quantmod)
+library(magrittr)
+library(dash)
+library(dashCoreComponents)
+library(dashHtmlComponents)
+library(glue)
+library(jsonlite)
+library(rlang)
+
```

Comment:
why so many libraries? Cut some you never use. The less packages you depend on, the more reliable your code gotta be.

Reply:
`rlang`:: <- to use `is_empty()`
the other two get installed for `dash` and `plotly` anyways. :)